var/spack/repos/builtin/packages/py-snakemake-storage-plugin-s3/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+
+from spack.package import *
+
+
+class PySnakemakeStoragePluginS3(PythonPackage):
+    """A Snakemake storage plugin for S3 API storage (AWS S3, MinIO, etc.)."""
+
+    pypi = "snakemake_storage_plugin_s3/snakemake_storage_plugin_s3-0.2.9.tar.gz"
+
+    license("MIT")
+
+    homepage = "https://github.com/snakemake/snakemake-storage-plugin-s3"
```

Comment:
We usually put homepage before url/pypi/git

Reply:
Reordered